test/test-files/swift/swift-package-project-with-confjg/.swift-format

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
There is a typo in the folder path:   __with-confjg__ ->  __with-config__ this is why the test does not work.